src/components/PastTours/PastTourDetails.tsx

```diff
@@ -53,15 +53,15 @@ export const PastTourDetails: React.FC<RouteComponentProps<TParams>> = ({
   )
 
   return (
-    <section className="tour-details">
+    <main className="tour-details">
       <div className="location">
         <h1>Buffalo Mountain</h1>
         <p className="date">Feb 2, 2021</p>
       </div>
 
       <section className="plan">
         <div className="title-wrapper">
-          <img src={lightbulb} alt="lightbulb" />
+          <img src={lightbulb} alt="lightbulb" className="form-icon" />
```

Comment:
Thanks for adding these icons here, this is way more consistent to look like the past tour details forms!